sdk/storage/Azure.Storage.Common/src/GeoRedundantReadPolicy.cs

```diff
@@ -32,27 +33,27 @@ public override void OnSendingRequest(HttpPipelineMessage message)
 
             // Look up what the alternate host is set to in the message properties. For the initial request, this will
             // not be set.
-            var alternateHost =
+            string alternateHost =
                 message.TryGetProperty(
-                    Constants.GeoRedundantReadConstants.AlternateHostKey,
+                    Constants.GeoRedundantRead.AlternateHostKey,
                     out var alternateHostObj)
                 ? alternateHostObj as string
                 : null;
             if (alternateHost == null)
             {
                 // queue up the secondary host for subsequent retries
-                message.SetProperty(Constants.GeoRedundantReadConstants.AlternateHostKey, this._secondaryStorageHost);
+                message.SetProperty(Constants.GeoRedundantRead.AlternateHostKey, this._secondaryStorageHost);
                 return;
             }
 
             // Check the flag that indicates whether the resource has not been propagated to the secondary host yet.
             // If this flag is set, we don't want to retry against the secondary host again for any subsequent retries.
             // Also, the flag being set implies that the current request must already be set to the primary host, so we
             // are safe to return without checking if the current host is secondary or primary.
-            var resourceNotReplicatedToSecondaryStorage =
-                message.TryGetProperty(Constants.GeoRedundantReadConstants.ResourceNotReplicatedToSecondaryStorageKey, out var value)
+            var resourceNotReplicated =
+                message.TryGetProperty(Constants.GeoRedundantRead.ResourceNotReplicatedToSecondaryStorageKey, out var value)
```

Comment:
I meant `ResourceNotReplicatedToSecondaryStorageKey` specifically, but `resourceNotReplicatedToSecondaryStorage` was good to shorten too.  😄